Assets/Plugins/Android/Editor/PlatformSpecificConfigEditorAndroid.cs

```diff
@@ -69,29 +46,24 @@ public void OnGUI()
                 overrideValues = configFile.currentEOSConfig.overrideValues;
             }
 
-            EpicOnlineServicesConfigEditor.AssigningFlagTextField("Override Platform Flags (Seperated by '|')", ref overrideValues.platformOptionsFlags, 250);
+            GUIEditorHelper.AssigningFlagTextField("Override Platform Flags (Seperated by '|')", ref overrideValues.platformOptionsFlags, 250);
 
-            EpicOnlineServicesConfigEditor.AssigningFloatToStringField("Override initial button delay for overlay", ref overrideValues.initialButtonDelayForOverlay, 250);
+            GUIEditorHelper.AssigningFloatToStringField("Override initial button delay for overlay", ref overrideValues.initialButtonDelayForOverlay, 250);
 
-            EpicOnlineServicesConfigEditor.AssigningFloatToStringField("Override repeat button delay for overlay", ref overrideValues.repeatButtonDelayForOverlay, 250);
+            GUIEditorHelper.AssigningFloatToStringField("Override repeat button delay for overlay", ref overrideValues.repeatButtonDelayForOverlay, 250);
```

Comment:
Many calls to `EpicOnlineServicesConfigEditor` were being used across all the `EditorWindow` classes to draw GUI controls. Instead of having one `EditorWindow` derived class contain all these methods, I moved them to a new static class `GUIEditorHelper`.

Reply:
Great call, makes total sense to separate these.